compiler/codegen.js

```diff
@@ -5423,6 +5425,28 @@ const internalConstrs = {
     type: TYPES.undefined,
     notConstr: true,
     length: 1
+  },
+
+  __Porffor_rawType: {
+    generate: (scope, decl) => {
+      const type = getNodeType(scope, decl.arguments[0]);
+      if (type) {
+        return [
+          ...type,
+          Opcodes.i32_from
+        ];
+      }
+      return generate(scope, {
```

Comment:
nit: add blank line here and use single quotes

Reply:
istg one day i'll remember to use single quotes lmao